tests/test_event_updates.py

```diff
@@ -12,6 +13,55 @@
     get_catalogs, update_object, update_payment_intent, create_object, delete_object
 
 
+class TestEventUpdatesSyncStart(BaseTapTest):
```

Comment:
Add test cases for the Event creation scenario

Reply:
Added `Events` stream in to cover in the tap-tester.